doc/specs/staffPage.md

```diff
@@ -0,0 +1,161 @@
+# Staff Page
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+4. Data model
+
+## Motivation
+
+>The staff team is the core of the smooth operation of the site. The staff page is a list of the site's staff members for the user's convenience.
+
+## Business Rules
+
+> * User classes must have an option to be presented on the staff page.
+> * Users with the class management permission must be able to toggle if a user class is displayed on the staff page.
+> * User classes with dedicated staff page descriptions must get their own sections on the staff page (i.e. forum moderators).
+
+## Use Cases
+
+### 1.0 A user visits the staff page
+
+**Primary Actor:** A user
+
+**Brief:** Allows users to see the staff page.
+
+**Preconditions:** N/A
+
+**Trigger:** A user navigates to the staff page.
+
+**Postconditions:** The staff page is rendered.
+
+**Basic Flow:**
+
+1. The user navigates to the staff page.
+2. The staff page is generated, including the username hyperlink and the user's staff page description.
+### 2.0 User editing their staff page description
+
+**Primary Actor:** A user that qualifies for a staff page description
+
+**Brief:** Staff members must be able to edit the staff page description that shows up next to their username on the staff page.
+
+**Precondition:** The user is in a class that is displayed on the staff page.
+
+**Trigger:** The user clicks submit on their profile edit page.
+
+**Postconditions:** The user's staff page description is updated.
+
+**Basic Flow:**
+
+1. The user navigates to their user edit page.
+2. The user enters a new staff page description.
+3. The user clicks submit, and the staff page description is saved.
+
+**Functional Requirements:**
+
+* The staff page description must be saved to the database.
+
+### 3.0 Adding a user class to the staff page
+
+**Primary Actor:** A user with the class management permission
+
+**Brief:** User classes must be able to be added and removed from the staff page.
```

Comment:
Do you mean that someone can edit *my* classes on the staff page, or edit the classes which are shown on the staff page?